src/utilities.rs

```diff
@@ -42,6 +75,18 @@ impl Node {
     }
 }
 
+impl Distribution<Node> for Standard {
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Node {
+        Node(rng.gen())
+    }
+}
+
+impl Debug for Node {
+    fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
```

Comment:
I'd rather not change these, we can pretty much copy paste from the console output to the code right now, when fixing test cases.

Reply:
Hmm - I don't mind keeping a local copy for that one big change (`MemberState`), but I really don't want to have to make a lot of changes locally before I can even really start debugging effectively.

How about if I add these as `Display` traits instead?

Reply:
I was wondering if your MemberState printing would work without it?